### DISCUSSION TITLE
feat: Folder structure for dataset organisation

### DISCUSSION THREAD

**Discussion by marliessophie** [langfuse/langfuse#3935] [Ideas]
### Describe the feature or potential improvement

As we add more datasets, it would be helpful to organise them in folders.

**Comment by marcklingen** [comment]
Great idea. In the meantime, tagging datasets can help to keep them organized.

**Comment by jakekang-cloudcruise** [comment]
+1 on the folders for datasets! is tagging or folders on the roadmap yet?

**Comment by harmoney-andrewb** [comment]
Tagging useful - but having folder structure aligns better with Organisational Structures - so having both would be useful!

**Comment by dahnny012** [comment]
do we know if this is a big change? I will be generating a lot of datasets so organizing them as a folder is essential. I can help contribute a PR if no one is working on it.

**Comment by jannikmaierhoefer** [comment]
Hi all, folders for datasets are now supported: https://langfuse.com/changelog/2025-10-27-dataset-folders

Please let me know if you have any feedback!
